lib/bpk-component-calendar/stories.js

```diff
@@ -288,6 +289,7 @@ const CalendarWithPicker = (props: CalendarWithPickerProps) => {
 
 const ChangeableSelectionTypeStory = () => {
   const options = Object.keys(SELECTION_TYPES).reduce(
+    // $FlowFixMe - Literals probably should fix it.
```

Comment:
Given this component is bridged I decided to just add this as I was not sure how to fix this but we should fix it